backend/package.json

```diff
@@ -17,7 +18,7 @@
   },
   "dependencies": {
     "@relay-vaults/abis": "workspace:*",
-    "@relay-vaults/addresses": "1.0.5",
+    "@relay-vaults/addresses": "1.1.0",
```

Comment:
I think we need to check if this actually works (installs only the npm version and not the "local" version). If not let's just use the local version?